docs/cicd/others-script-cd-template.sh

```diff
@@ -0,0 +1,64 @@
+# This is just an example workflow for continuous deployment.
```

Comment:
bash files need to start with `#!/usr/bin/env bash` and then you need `set -euxo pipefail` or else the script won't halt execution if `npm run test` returns nonzero

Reply:
technically you don't need `set -x` but it's good practice because the logs will show what commands are being executed. WARNING when you have random secrets in ENV variables, they will get printed to stdout if they are being expanded in any variables like $VAR, this is another reason why passwords in ENV are a bad idea. (glad we are at least recommending encryption with GH secrets)

Reply:
updated